src/main/java/com/kodilla/ecommercee/User.java

```diff
@@ -0,0 +1,98 @@
+package com.kodilla.ecommercee;
+
+import com.kodilla.ecommercee.temp.Cart;
+import com.kodilla.ecommercee.temp.Order;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@Table(name = "users")
+public class User {
+    private long id;
+    private String name;
+    private String surname;
+    private String mail;
+    private String password;
```

Comment:
dodałeś nowe pola, ale pominąłeś te które są w zadaniu - private Boolean isBlocked oraz private String token;

Reply:
O, to dodam, dzięki